include/mixer.h

```diff
@@ -420,6 +420,7 @@ void MIXER_DeregisterChannel(mixer_channel_t& channel);
 // Mixer configuration and initialization
 void MIXER_AddConfigSection(const config_ptr_t &conf);
 int MIXER_GetSampleRate();
+uint8_t MIXER_GetPreBufferMs();
```

Comment:
Isn't that value-range limitation a bit too strict? (0-255) I agree pre-buffer values should be low, but maybe relax it to 16 bits?

Reply:
Our current 16 KB sample buffer isn't big enough (and I didn't want to get into re-factoring mixer buffers and such, in this PR). 

``` c++
#define MIXER_BUFSIZE (16 * 1024)
#define MIXER_BUFMASK (MIXER_BUFSIZE - 1)
alignas(sizeof(float)) uint8_t MixTemp[MIXER_BUFSIZE] = {};
```

But yeah - we should probably move to a queue that's sized based on the desired latency and playback rate.

I've also set the maximum-selectable latency to 100 ms, just because at that point, the experience in games is going to be quite bad, in terms of sound effects come quite a bit after the action happens.

I'll bump the return type just so it's not a blocker in the future - but will leave the internals and the size limits.

Reply:
The 100 ms limit sound reasonable, I agree. I guess I was just thinking not to restrict this too much unnecessarily if there's not much to gain by it, plus it avoids nasty surprises later if we'd need to go above 255 for whatever reason.